vault-core/src/main/java/org/apache/jackrabbit/vault/fs/impl/io/DocViewImporter.java

```diff
@@ -1200,28 +1200,37 @@ private boolean isPropertyProtected(@NotNull EffectiveNodeType effectiveNodeType
         return effectiveNodeType.getApplicablePropertyDefinition(npResolver.getJCRName(docViewProperty.getName()), docViewProperty.isMultiValue(), docViewProperty.getType()).map(PropertyDefinition::isProtected).orElse(false);
     }
 
+    private String nameToPathComponent(Name name) throws RepositoryException {
```

Comment:
maybe `getLexicalForm` as method name (https://developer.adobe.com/experience-manager/reference-materials/spec/jcr/2.0/3_Repository_Model.html#3.2.5%20Lexical%20Form%20of%20JCR%20Names) which would allow both expanded and qualified form.

Reply:
sounds right to me